src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -94,5 +96,15 @@ public HostStatus GetHostStatus()
 
             return status;
         }
+
+        public override Task<HttpResponseMessage> ExecuteAsync(HttpControllerContext controllerContext, CancellationToken cancellationToken)
+        {
+            // For all admin api requests, we'll update the ScriptHost debug timeout
+            // For now, we'll enable debug mode on ANY admin requests. Since the Portal interacts through
+            // the admin API this is sufficient for identifying when the Portal is connected.
+            _scriptHostManager.Instance.NotifyDebug();
```

Comment:
No need for a special portal header right now. We may change this in the future, but for now ANY admin API traffic puts the host in debug mode.